test/jdk/java/lang/reflect/exeCallerAccessTest/CallerAccessTest.java

```diff
@@ -50,10 +49,8 @@ public static void main(String[] args) throws IOException {
         ProcessBuilder pb = new ProcessBuilder(launcher.toString());
         Map<String, String> env = pb.environment();
 
-        String libName = Platform.isWindows() ? "bin" : "lib";
-        Path libPath = Paths.get(Utils.TEST_JDK).resolve(libName);
-        String libDir = libPath.toAbsolutePath().toString();
-        String serverDir = libPath.resolve("server").toAbsolutePath().toString();
+        String libDir = Platform.libDir().toString();
+        String serverDir = Platform.jvmLibDir().toString();
```

Comment:
Perhaps `vmDir` would be a more suitable name?

Reply:
Makes sense, thank you.